neutronclient/client.py

```diff
@@ -239,6 +248,8 @@ def authenticate(self):
             self._authenticate_keystone()
         elif self.auth_strategy == 'noauth':
             self._authenticate_noauth()
+        elif self.auth_plugin:
+            self._plugin_auth(self.auth_url)
```

Comment:
I mean, yeah if someone has the plugin installed and they have no idea they did it, well then yeah that'd suck (but that's protecting a user who is already pretty tech savvy). But if they wanted to override they would just pass in keystone.

Just seems silly to install a plugin to purposefully deviate from the norm and then change all your scripts to use it.

I leave it to your judgement @mpath